<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SnopfManager</class>
 <widget class="QMainWindow" name="SnopfManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Snopf</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/icon/icon.svg</normaloff>:/icon/icon/icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="accountTableView">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionSaveChanges"/>
   </widget>
   <widget class="QMenu" name="menuEntries">
    <property name="title">
     <string>E&amp;ntries</string>
    </property>
    <addaction name="actionNewEntry"/>
    <addaction name="actionDeleteEntry"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
    <addaction name="actionVersionInfo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEntries"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenAccountTable">
   <property name="text">
    <string>&amp;Open Account Table</string>
   </property>
  </action>
  <action name="actionSetCurrentFileAsStandard">
   <property name="text">
    <string>&amp;Set Current File as Default</string>
   </property>
  </action>
  <action name="actionNewEntry">
   <property name="text">
    <string>&amp;New Entry</string>
   </property>
  </action>
  <action name="actionDeleteEntry">
   <property name="text">
    <string>&amp;Delete Entry</string>
   </property>
  </action>
  <action name="actionSaveChanges">
   <property name="text">
    <string>Save &amp;Changes</string>
   </property>
  </action>
  <action name="actionVersionInfo">
   <property name="text">
    <string>&amp;Version Info</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
